package.json

```diff
@@ -32,8 +32,8 @@
     "gh-pages": "2.0.0",
     "gulp-htmllint": "0.0.16",
     "jest": "^27.3.1",
-    "node-sass": "^4.14.1",
     "parcel": "^1.12.4",
+    "sass": "*",
     "stylelint": "^13.5.0",
     "stylelint-config-recommended-scss": "^3.3.0",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.

Reply:
Hello,without package.json, deploy isnt working